network/containerizer/bridgepolicy.go

```diff
@@ -230,7 +237,7 @@ func (b *BridgePolicy) FindMissingBridgesForContainer(m Machine, containerMachin
 	}
 	notFound := containerSpaces.Difference(spacesFound)
 	fanNotFound := containerSpaces.Difference(fanSpacesFound)
-	if b.ContainerNetworkingMethod == "fan" {
+	if p.ContainerNetworkingMethod == "fan" {
```

Comment:
Just curious: do we define any consts for things like "fan", "local" etc. in corenetwork that we could use instead of hardcoding values?

Reply:
Doesn't look like we do at the moment. I will look at doing it with a future patch.